decidim-core/spec/lib/attribute_object/form_spec.rb

```diff
@@ -299,7 +299,7 @@ def custom_sauce
     describe ".hash_from" do
       subject { described_class.hash_from(value) }
 
-      let(:hash) { { foo: "bar", "baz" => "biz" } }
+      let(:hash) { { :foo => "bar", "baz" => "biz" } }
```

Comment:
I am not 100% sure why we are using 2 types of hash here...

Reply:
I actually think that in this case it was intended: as we're accepting hashes from clients of this class, we actually want to try that both of the syntaxes work (even when they're mixed). See that the spec is [checking it out with indifferent access](https://github.com/decidim/decidim/blob/3bcb524f420ed15d29a5a0c81ff9efb88071af06/decidim-core/spec/lib/attribute_object/form_spec.rb#L305)